README.adoc

```diff
@@ -60,7 +60,7 @@ If you use link:http://nixos.org/nix[Nix] you may execute the `nix-shell` comman
 How to run `emerald` (by default on port '1920'):
 
 ----
-$ emerald server
+$ target/debug/emerald server
```

Comment:
This is not a case, you can simply run `emerald server` after `cargo install emerald-cli` as stated on the top of doc